ci/docker/tizen/prepare-sync.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash -e
+
+DEPS=src/flutter/DEPS
+
+if [ ! -f $DEPS ]; then
+    echo "Could not find DEPS file."
+    exit 1
+fi
+
+cat > .gclient << EOF
+solutions = [
+  { "name"        : 'src/flutter',
+    "url"         : 'https://github.com/flutter-tizen/engine.git',
+    "deps_file"   : 'DEPS',
+    "managed"     : False,
+    "custom_deps" : {
+    },
+    "custom_vars": {},
+  },
+]
+EOF
```

Comment:
Why don't use use a `gclient config` command like this:

```sh
gclient config --name="src/flutter" --deps-file="DEPS" --unmanaged https://github.com/flutter-tizen/engine.git
```

Reply:
`gclient config` is also ok. but `gclient sync` seems to do other things that we don't need. sometimes it takes more than 10 seconds.